components/gitpod-db/src/typeorm/migration/1699012261520-WorkspaceClusterAvailableClasses.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { MigrationInterface, QueryRunner } from "typeorm";
+import { columnExists } from "./helper/helper";
+
+const table = "d_b_workspace_cluster";
+const columnAWC = "availableWorkspaceClasses";
+const columnPrefClass = "preferredWorkspaceClass";
+
+export class WorkspaceClusterAvailableClasses1699012261520 implements MigrationInterface {
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        if (!(await columnExists(queryRunner, table, columnAWC))) {
+            await queryRunner.query(`ALTER TABLE ${table} ADD COLUMN ${columnAWC} TEXT, ALGORITHM=INPLACE, LOCK=NONE`);
```

Comment:
use `JSON`

Reply:
We use TEXT in most other JSON cases. Would prefer to keep it this way for now, until we decide to migrate/make a concerted effort to change this. 
(consistency over point improvements).